templates/base/package.json

```diff
@@ -28,17 +28,20 @@
   ],
   "dependencies": {
     "@apollo/react-hooks": "^3.1.5",
+    "@mdx-js/react": "^1.6.22",
+    "@storybook/addon-docs": "^6.3.8",
```

Comment:
Story book will have its own package.json, packages related to it should not be part of base or arch type